otx/algorithms/segmentation/adapters/mmseg/data/dataset.py

```diff
@@ -263,6 +263,10 @@ def __init__(self, **kwargs):
         else:
             classes = []
         super().__init__(otx_dataset=otx_dataset, pipeline=pipeline, classes=classes)
+
+        # TODO [Soobee] : Need to align again when background label is removed
+        self.CLASSES = ["background"] + [label.name for label in self.project_labels]
```

Comment:
I think it would be nice to add some code here to check if this has a class called `background`, what do you think?

Reply:
Good. will revise it.

Reply:
IMO, we need to consider the variants such as `Background` or `back ground`...